.github/workflows/ci.yml

```diff
@@ -12,8 +12,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: ["2.7", "3.0", "3.1", "3.2", "3.3", "3.4", "ruby-head"]
-        sidekiq: ["~> 6", "~> 7"]
+        ruby: ["3.2", "3.3", "3.4", "ruby-head"]
```

Comment:
I'd like to keep compatibility with those Rubies, for people still on Sidekiq v6 or v7 it makes sense.